sonarqube/tests/test_unit_components.py

```diff
@@ -0,0 +1,382 @@
+import re
+from random import randrange
+
+import mock
+import pytest
+
+from .common import HOST, METRICS, PORT
+
+pytestmark = [pytest.mark.unit]
+
+
+@mock.patch("datadog_checks.sonarqube.check.SonarqubeAPI")
+def test_service_check_ok(mock_api, aggregator, dd_run_check, sonarqube_check):
+    # Given
+    config = {
```

Comment:
While I agree that `config` is a clearer name overall, the reality of our codebase is that we use `instance` for this type of object, so let's stick to that name.